PetAdoption-iOS/PetAdoption-iOS/Controllers/PetListingViewController.swift

```diff
@@ -237,15 +261,16 @@ extension PetListingViewController : UICollectionViewDelegate, UICollectionViewD
 {
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int
     {
-        return self.petData.count
+        self.noResultsLabel.isHidden = !(self.filteredPetData.count == 0)
+        return self.filteredPetData.count
     }
 
     ////////////////////////////////////////////////////////////
 
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell
     {
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: PetCell.reuseIdentifier, for: indexPath) as! PetCell
-        let pet = petData[indexPath.row]
+        let pet = filteredPetData[indexPath.row]//petData[indexPath.row]
```

Comment:
You can just remove the commented out code at the end of this line.  Unless you left it there for a reason.

Reply:
Removed with d39f3a1